modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseUpgradeCheck.java

```diff
@@ -6,13 +6,36 @@
 package com.liferay.source.formatter.check;
 
 import com.liferay.petra.string.CharPool;
+import com.liferay.portal.kernel.util.ArrayUtil;
 import com.liferay.source.formatter.check.util.JavaSourceUtil;
 
+import java.util.List;
+
 /**
  * @author Nícolas Moura
  */
 public abstract class BaseUpgradeCheck extends BaseFileCheck {
 
+	public boolean validateParameters(
```

Comment:
@TamyrisTorres, let's rename to `hasValidParameters` instead to keep consistency with the recent methods created by us, for example `hasParameterTypes`.